api/engineapi/cli.py

```diff
@@ -444,6 +420,15 @@ def delete_user_handler(args: argparse.Namespace) -> None:
     pass
 
 
+def sign_handler(args: argparse.Namespace) -> None:
+    # Prompt user to enter the password for their signing account
+    password_raw = input(f"Enter password for signing account ({args.signer}): ")
+    password = password_raw.strip()
```

Comment:
Why we stopped using signatures here, for example I would prefer to sign message on local machine and pass it on production if required admin access?

Reply:
I don't understand the question. We didn't stop using signatures - the schema for `dropper-v0.2.0` requires a signature.